docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md

```diff
@@ -44,18 +43,21 @@ To convert a model to OpenVINO model format (``ov.Model``), you can use the foll
 
 If the out-of-the-box conversion (only the ``input_model`` parameter is specified) is not successful, use the parameters mentioned below to override input shapes and cut the model:
 
-- model conversion API provides two parameters to override original input shapes for model conversion: ``input`` and ``input_shape``.
-  For more information about these parameters, refer to the :doc:`Setting Input Shapes <openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model>` guide.
+- ``input`` and ``input_shape`` - the model conversion API parameters used to override original input shapes for model conversion,
+
+  For more information about the parameters, refer to the :doc:`Setting Input Shapes <openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model>` guide.
+
+- ``input`` and ``output`` - the model conversion API parameters used to define new inputs and outputs of the converted model to cut off unwanted parts (such as unsupported operations and training sub-graphs),
 
-- To cut off unwanted parts of a model (such as unsupported operations and training sub-graphs),
-  use the ``input`` and ``output`` parameters to define new inputs and outputs of the converted model.
   For a more detailed description, refer to the :doc:`Cutting Off Parts of a Model <openvino_docs_MO_DG_prepare_model_convert_model_Cutting_Model>` guide.
 
-You can also insert additional input pre-processing sub-graphs into the converted model by using
-the ``mean_values``, ``scales_values``, ``layout``, and other parameters described
-in the :doc:`Embedding Preprocessing Computation <openvino_docs_MO_DG_Additional_Optimization_Use_Cases>` article.
+- ``mean_values``, ``scales_values``, ``layout`` - the parameters used to insert additional input pre-processing sub-graphs into the converted model,
+
+  For more details, see the :doc:`Embedding Preprocessing Computation <openvino_docs_MO_DG_Additional_Optimization_Use_Cases>` article.
+
+- ``compress_to_fp16`` - a compression parameter in ``mo`` command-line tool, which allows generating IR with constants (for example, weights for convolutions and matrix multiplications) compressed to ``FP16`` data type.
 
-The ``compress_to_fp16`` compression parameter in ``mo`` command-line tool allows generating IR with constants (for example, weights for convolutions and matrix multiplications) compressed to ``FP16`` data type. For more details, refer to the :doc:`Compression of a Model to FP16 <openvino_docs_MO_DG_FP16_Compression>` guide.
```

Comment:
why is this `The ``compress_to_fp16`` compression parameter in ``mo`` command-line tool allows generating IR with constants (for example, weights for convolutions and matrix multiplications) compressed to ``FP16`` data type.` removed?

Reply:
It is not removed, it's been redacted and it is still there, line 58.